scalding-parquet/src/main/scala/com/twitter/scalding/parquet/tuple/macros/Macros.scala

```diff
@@ -0,0 +1,50 @@
+package com.twitter.scalding.parquet.tuple.macros
+
+import com.twitter.scalding.parquet.tuple.macros.impl.{ ParquetSchemaProvider, ParquetTupleConverterProvider, WriteSupportProvider }
+import com.twitter.scalding.parquet.tuple.scheme.ParquetTupleConverter
+import parquet.io.api.RecordConsumer
+import parquet.schema.MessageType
+
+import scala.language.experimental.macros
+
+object Macros {
```

Comment:
none of those macros depend on scalding and could be in the parquet project.
You could use them with Spark for example.